_inc/lib/admin-pages/class.jetpack-react-page.php

```diff
@@ -302,7 +302,7 @@ function page_admin_scripts() {
 				 * @param bool $are_promotions_active Status of promotions visibility. True by default.
 				 */
 				'showPromotions' => apply_filters( 'jetpack_show_promotions', true ),
-				'isAutomatedTransfer' => jetpack_is_automated_transfer_site(),
+				'isAutomatedTransfer' => jetpack_is_atomic_site(),
```

Comment:
Minor point, but should we also update this to `isAtomicSite`?

Reply:
Good point! Yes, we should, hopefully there are not a lot of references to that one either.

Reply:
Tried to do that 28571f01f65c756ba5db80643034153d4d95bcce, requires further inspection/testing.